osconfig_tests/test_config/project_config.go

```diff
@@ -0,0 +1,45 @@
+///  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package testconfig
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/cloudresourcemanager"
+)
+
+type ProjectConfig struct {
+	TestProjectID        string
+	TestProjectNumber    string
+	ServiceAccountEmail  string
+	TestZone             string
+	ServiceAccountScopes []string
+}
+
+func New(tpid, tz string, sass []string) (*ProjectConfig, error) {
+	tpn, err := cloudresourcemanager.GetProjectNumberFromProjectID(tpid)
+	if err != nil {
+		fmt.Printf("%+v", err)
+		return nil, err
+	}
+
+	return &ProjectConfig{
+		TestProjectID:        tpid,
+		TestProjectNumber:    tpn,
+		ServiceAccountEmail:  fmt.Sprintf("%s-compute@developer.gserviceaccount.com", tpn),
```

Comment:
just use "default", no need to do any of the GetProjectNumberFromProjectID stuff as the osconfig api can take ProjectID just fine. I would just remove this package completly as it's not doing much once you remove the lookup and it's not universally useful accros tests.

Reply:
yeah, the reason i introduced the cloudresourcemanager package was because i was under the impression that projectID is not allowed for api calls.
however, i would still insist on keeping the projectconfig struct so that all the project related configurations are clubbed together.
I am thinking of moving it to osconfig_tests/main and so the projectconfig can be created once and used in all the test suites( i could have done this before sending the PR itself)

Reply:
Do you still want the project number? It'd be better to use the compute client your already have (GetProject) instead of invoking yet another api client.

Reply:
i dont need the projectNumber, will remove it from the struct. but we still need the projectid, serviceaccount, and other information

Reply:
ok.. instead of moving it to main package, i thought the best place for the project info is the compute package.